yaksh/bash_code_evaluator.py

```diff
@@ -133,10 +136,11 @@ def check_code(self, user_answer, file_paths, test_case):
                     proc, stdnt_stdout, stdnt_stderr = ret
                     valid_answer = inst_stdout == stdnt_stdout
             if valid_answer and (num_lines == loop_count):
-                return True, "Correct answer"
+                test_case_weightage = float(weightage) if partial_grading else 0.0
+                return True, "Correct answer", test_case_weightage
             else:
                 err = ("Error:expected"
                     " {0}, got {1}").format(inst_stdout+inst_stderr,
                         stdnt_stdout+stdnt_stderr
                     )
-                return False, err
+                return False, err, test_case_weightage
```

Comment:
I have initialised the test_case_weightage as 0.0 earlier. Hence the variable is returned as is. 

However I agree that this may be a bit unclear, hence fixing this to return a constant 0.0